tests/unit/amo/components/TestAddonsCard.js

```diff
@@ -166,6 +166,18 @@ describe(__filename, () => {
     expect(root.find(CardList)).toHaveProp('header', header);
   });
 
+  it.each([true, false])(
```

Comment:
I missed this last time, but as with the test for `LandingAddonsCard`, we don't really need to test both `true` and `false` here. I'd just test with `true`.

Reply:
Ah, I had updated the wrong test - sorry! Updated!